modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -1837,13 +1874,6 @@ func (suite *KeeperTestSuite) TestAbortHandshake() {
 			},
 			expPass: true,
 		},
-		{
```

Comment:
how come this test was removed?

Reply:
The upgrade is no longer fetched from within `MustAbortHandshake`, it was only used to access the fields which were emitted in the events which were removed.

Reply:
cool, sounds good! Should we rename this test func, it threw me off. 

Function is called `AbortUpgrade` / `MustAbortUpgrade` and test name is `TestAbortHandshake`